io-pay-portal-fe/src/api/__tests__/services.test.ts

```diff
@@ -0,0 +1,5 @@
+describe("withApiRequestWrapper", () => {
+  it("should return a ", async () => {
```

Comment:
Obviously this is just a stub test to develop the CI/CD pipelines. Can you confirm?